### ISSUE TITLE
BUG - KSPWheelBrakes not showing up in 'Brakes' Action-Group nor is it assignable 

### ISSUE THREAD

**Issue by hippomormor** [shadowmage45/KSPWheel#15]
Brakes are not showing in the "Brakes" action-group in the SPH editor nor is it available to assign. Brake functionality seems to be missing on all wheels included in Foundries including the ALG-wheels.
Reproduction:
Place any Foundries wheel, check the 'Brake' action-group - it will be empty.
Click any Foundries wheel - Brakes will not be available to be assigned as an Action.
Please tell if you need a log - didn't attach any since this seems to be a minor typo/mistake somewhere..

**Comment by shadowmage45** [comment]
Why would it matter if it shows up in the brakes AG if it is not reassignable anyway?

(The stock brake action group on wheels is not reassignable, is hard-coded to use the brakes AG.  Edit -- or is it?  I've never tried messing with the stock wheels/brakes setup... )

**Comment by hippomormor** [comment]
I think you misunderstood me. Let me rephrase it - Brakes doesn't work on any wheels from Foundries. Pressing the Brake hotkey/GUI button has no effect in-game. 
And all the stock landing gear do show the Brake-action in the Brake action group - and they can be removed and reassigned. Foundries wheels show nothing in the action group, and the brake button seems to do nothing.
https://www.dropbox.com/s/9jla1rseqlgf49q/Capture.PNG
https://www.dropbox.com/s/hehgqhlvaun2hia/Capture2.PNG
It's not that it matters if they can be reassigned or shows up in the AG - It's more that it doesn't brake in-game

**Comment by shadowmage45** [comment]
Apparently there was a message in here somewhere that got deleted:

>I think you misunderstood me. Let me rephrase it - Brakes doesn't work on any wheels from Foundries. Pressing the Brake hotkey/GUI button has no effect in-game.
And all the stock landing gear do show the Brake-action in the Brake action group - and they can be removed and reassigned. Foundries wheels show nothing in the action group, and the brake button seems to do nothing.
https://www.dropbox.com/s/9jla1rseqlgf49q/Capture.PNG
https://www.dropbox.com/s/hehgqhlvaun2hia/Capture2.PNG
It's not that it matters if they can be reassigned or shows up in the AG - It's more that it doesn't brake in-game

If the brakes are -not working- for you, that would be a separate bug, that needs its own bug report, with logs.  As I can assure you the brakes are working fine in my dev setup (press either the 'b' key or the brake button near the altimiter).

(commit note not directed at you; is referring to how silly the stock action group code setup is....)

**Comment by hippomormor** [comment]
Ok ok ok.. Removed all mods and did a test with a small plane and it worked fine. It was probably because the airplane was to big or had the engines in idle pushing it forward, i don't know.. The discrepancy between stock and Foundries wheels in the AG made me believe it wasn't working, my bad :/

**Comment by shadowmage45** [comment]
No worries, I fully understand how confusing tracking down seeming problems can be (I've spent hours trying to find the cause of an issue, only to find it was simple config typo...).  Just wanted to make sure there wasn't a secondary issue here that needed to be fixed as well.

If you do happen to spot some sort of mod-conflict that was keeping the brakes from working, please let me know on that as well.  Mod conflicts are something I can generally resolve one way or another, as long as I know there is a conflict, and with which mod(s).

On a good note -- I -think- that I have it setup so that the brakes will show up in action groups and be re-assignable.  Should be available with the next update.

**Comment by hippomormor** [comment]
Further testing shows everything seems to work the same with mods re-added (FAR, KJR, IR etc.). Breaking seems to work normally.

Thanks a lot for your support, both regarding my FAR issue and this! Must say you are a very committed dev.

Yeah.. saw the commit :)
